Update expenses.md (#879)
flowforge/website: src/handbook/peopleops/expenses.md

## Expenses

> FlowForge will reimburse you for any work-related expenses that are considered reasonable. This includes providing you with the necessary equipment, such as a laptop, monitor, and accessories, to effectively carry out your job responsibilities. As an all-remote company, our meetings are conducted using either Zoom or Slack. Therefore, we will also cover the expenses associated with obtaining a high-quality webcam, microphone, and lighting setup.

Reviewer: ```suggestion
FlowForge will reimburse you for any work-related expenses that are considered reasonable. This includes providing you with the necessary equipment, such as a laptop, monitor, and accessories, to effectively carry out your job responsibilities. As an all-remote company, our meetings are conducted using either Google Meet or Slack. Therefore, we will also cover the expenses associated with obtaining a high-quality webcam, microphone, and lighting setup.
```

Reviewer: Reading this again, it feels like most of these lines should be moved to "Work & location" as section

> The general principle to follow is to treat work-related expenses as if they were your own money. It is important to seek good value for your purchases and use common sense when making decisions. If you find yourself in need of any equipment, it is advisable to have a discussion with your manager to explore the available options.

Reviewer: ```suggestion
The general principle to follow is to treat work-related expenses **as if they were your own money**. It is important to seek good value for your purchases and use common sense when making decisions. If you find yourself in need of any equipment, it is advisable to have a discussion with your manager to explore the available options.
```

> There are two methods available for using company funds: utilizing a company credit card or seeking reimbursement. If employees opt to use Brex for purchasing any company-related items or subscriptions, it is essential to upload the receipts and include a memo within three business days. For all reimbursement requests, please ensure to submit the required documentation and proofs through Deel.

Reviewer: Subscriptions need approval, as vendors require approval. @knolleary is working on a list of currently approved vendors. We should link that here.

### Workspace & Location

> As an all-remote company without physical offices, the expectation is that most individuals will primarily work from their homes. When it comes to acquiring equipment and furniture for your home workspace, it is recommended to discuss these requirements with your manager during the hiring process. It's important to note that the company's coverage of certain costs, such as broadband expenses, may have tax implications depending on the local legislation in your area. To understand the specific tax implications relevant to your situation, it is advisable to consult an independent advisor familiar with the local tax regulations in your location.

Reviewer: We don't offer broadband currently, nor is it planned. It's a legal minefield

```suggestion
As an all-remote company without physical offices, the expectation is that most individuals will primarily work from their homes. When it comes to acquiring equipment and furniture for your home workspace, it is recommended to discuss these requirements with your manager during the hiring process. It's important to note that the company's coverage of certain costs may have tax implications depending on the local legislation in your area. To understand the specific tax implications relevant to your situation, it is advisable to consult an independent advisor familiar with the local tax regulations in your location.
```
